Mathlib/GroupTheory/Perm/Basic.lean

```diff
@@ -587,7 +587,7 @@ theorem mul_swap_eq_iff {i j : α} {σ : Perm α} : σ * swap i j = σ ↔ i = j
 
 theorem swap_mul_swap_mul_swap {x y z : α} (hxy : x ≠ y) (hxz : x ≠ z) :
     swap y z * swap x y * swap y z = swap z x := by
-  nth_rewrite 2 [← swap_inv]
+  nth_rewrite 3 [← swap_inv]
```

Comment:
Did simprocs really make a difference here?

Reply:
No, this one is from leanprover/lean4#2539, which landed in [nightly-2024-01-03](https://github.com/leanprover/lean4-nightly/releases/tag/nightly-2024-01-03).

I thought we already had an adaptation PR for that, so perhaps this is a change that landed in Mathlib master and so was ad-hoc fixed on `nightly-testing` in the meantime.

I propose just rolling it in here to avoid the overhead of a separate PR.